lib/blocks.php

```diff
@@ -64,7 +64,9 @@ function gutenberg_reregister_core_block_types() {
 				'comment-edit-link.php'           => 'core/comment-edit-link',
 				'comment-reply-link.php'          => 'core/comment-reply-link',
 				'comment-template.php'            => 'core/comment-template',
+				'comments-pagination.php'         => 'core/comments-pagination',
```

Comment:
This one is already present in `trunk`. It might fix automatically after a rebase with `trunk`.